### DISCUSSION TITLE
Copilot not available in my region

### DISCUSSION THREAD

**Discussion by Ramtin2000** [community/community#51951] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Dear Github Team,

I'm writing to report an issue with Copilot that I'm experiencing from Istanbul, Turkey. Despite being located in Istanbul, Copilot is telling me that the service is not available in my country. This is preventing me from accessing the platform and using it for my work, which is causing a lot of frustration and inconvenience. I've checked that my account settings are correct, but the issue persists. Can you please investigate this issue and help me to resolve it? I would appreciate any assistance you can provide. Thank you for your attention to this matter.

Best regards,
Ramtin

**Comment by Akash1134** [comment]
Hi @Ramtin2000 ,

Sorry that you're getting Error 403 while using Copilot. I checked that your account does not have any restrictions and you should be getting this. Are you using any VPN service or have changed the settings in your account recently? Would you mind logging out of your account and checking if the issue persists?

**Reply by Ramtin2000** [reply]
@Akash1134 I can't even log back in now getting network errors 🗡️

**Reply by Lanrad23** [reply]
I ran into a similar issue before. Make sure you're not using a VPN or proxy that makes it look like you're in a different country where Copilot isn't available. This can confuse services about where you really are. For me, changing to a different VPN solved the problem.

**Comment by green122** [comment]
The same for my account, first location Bali Indonesia, the secon location now Malaisia, Kuala Lumpur

```
[INFO] [auth] [2023-04-04T13:45:26.699Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-04T13:45:26.704Z] GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"
```

Tried to re-logging, didn't help, tried VPN - the same :(

**Reply by hookenful** [reply]
The same, seem they started to flag accounts...

`[INFO] [auth] [2023-04-04T13:45:26.699Z] Invalid copilot token: missing token: 403 `

Did you find any workaround?

**Reply by green122** [reply]
Unfortunately no, It seems to be a problem with my account only inside our company. Firstly I thought it's  really because of the location but after I've changed the country and problem persists...

**Reply by green122** [reply]
Oh, it's started. Thank you guys if it's yours :)

**Reply by Akash1134** [reply]
Hi @green122 , We've made a few changes to your account and you should now be able to use Copilot. Thanks!

**Comment by Akash1134** [comment, marked as answer]
Hi @Ramtin2000 , could you logout of your account in IDE and Copilot, and log back in to see if it helps?

**Reply by Ramtin2000** [reply]
@Akash1134 Thanks it's working now what was the issue though?

**Reply by Akash1134** [reply]
There was a technical issue that we identified and fixed recently.

**Reply by RominaShekary** [reply]
Hello
I have the same problem too:(
Could you please help me?
Thanks

**Reply by OnurIlgaz** [reply]
Hi
I have the same problem too, could you please check my discussions
Thanks

**Comment by Mohammad78SD** [comment]
i have the same problem and when i try to use copilot it says:

`[INFO] [auth] [2023-10-30T08:08:47.148Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-10-30T08:08:47.150Z] Extension activation failed: "At this time, Copilot is not available in your location. You are currently logged in as Mohammad78SD."
[INFO] [auth] [2023-10-30T08:08:47.191Z] Invalid copilot token: missing token: 403 
`

what should i do to fix this issue? @Akash1134 can you please help me with this situation?

**Comment by saleem-alhabachi** [comment]
I have the same problem too from Istanbul, I hope you can solve it and thank you...
